acid-view/src/AcidView.hs

```diff
@@ -0,0 +1,45 @@
+module AcidView
+       (
+         loadState
+       , printDB
+       ) where
+
+import CmdParse
+import Universum
```

Comment:
You can use `loot-prelude` (or something like that, see in ariadne) to get universum implicitly imported anywhere.

Reply:
Thanks.